Update long-running.md (#1669)
demisto/content-docs: docs/integrations/long-running.md

![image](/doc_imgs/integrations/66122754-2d337b00-e5e9-11e9-8775-562e228e3fe6.png)

Certain integrations are long-running by default, they do not include the "Long Running Instance" checkbox. Examples of such integrations include:

- Generic Export Indicators Service
- Generic Webhook
- AWS SNS Listener
- TAXII2 Server
- TAXIIServer

In these cases, the long-running behavior is inherent and cannot be modified.
These integrations can operate in either "single engine" mode or "no engine" mode.

**Single Engine:** 

> - Select your engine in the integration's configuration.
> - When the engine is selected, a new field called "Listen Port" will appear.
> - You must specify the port number that the instance will use.

Reviewer: ```suggestion
1. Select your engine in the integration's configuration.
2. When the engine is selected, a new field called "Listen Port" will appear.
3. Specify the port number that the instance will use.
```

**No Engine:**

> - Select "no engine" in the integration configuration.
> - Click on "Save" and the server automatically assigns a port for the instance.

Reviewer: ```suggestion
1. Select "no engine" in the integration configuration.
2. Click **Save** and the server automatically assigns a port for the instance.
```

> Please note: If you select "No Engine", the listen port is assigned only after saving the configuration.

Reviewer: ```suggestion
Note: If you select **No Engine**, the listen port is assigned only after saving the configuration.
```

> You must click "Save" before clicking "Test" or performing any other actions with the instance.

Reviewer: ```suggestion
You must click **Save** before clicking **Test** or performing any other actions with the instance.
```

![image](/doc_imgs/integrations/66122754-2d337b00-e5e9-11e9-8775-562e228e3fe6.png)

Certain integrations are long-running by default, they do not include the "Long Running Instance" checkbox. Examples of such integrations include:

> - Generic Export Indicators Service

Reviewer 2: ```suggestion
- Generic Export Indicators Service (EDL)
```

- Generic Webhook
- AWS SNS Listener
- TAXII2 Server
- TAXIIServer

In these cases, the long-running behavior is inherent and cannot be modified.
These integrations can operate in either "single engine" mode or "no engine" mode.

**Single Engine:** 
1. Select your engine in the integration configuration.
2. When the engine is selected, a new field called "Listen Port" will appear.
3. Specify the port number that the instance will use.

**No Engine:**
1. Select "no engine" in the integration configuration.

> 2. Click **Save** and the server automatically assigns a port for the instance.

Reviewer 2: So this behavior is for the 8.8 server version right?
In 8.9, it is no longer the case; they can just test regularly.
So let's add a disclaimer for this, regarding the server versions

![image](/doc_imgs/integrations/66122754-2d337b00-e5e9-11e9-8775-562e228e3fe6.png)

> Certain integrations are long-running by default, they do not include the "Long Running Instance" checkbox. Examples of such integrations include:

Reviewer: ```suggestion
Certain integrations are long-running by default so they do not include the "Long Running Instance" checkbox. Examples of such integrations include:
```

- Generic Export Indicators Service (EDL)
- Generic Webhook
- AWS SNS Listener
- TAXII2 Server
- TAXIIServer

In these cases, the long-running behavior is inherent and cannot be modified.
These integrations can operate in either "single engine" mode or "no engine" mode.

**Single Engine:** 
1. Select your engine in the integration configuration.
2. When the engine is selected, a new field called "Listen Port" will appear.
3. Specify the port number that the instance will use.

**No Engine:**
1. Select "no engine" in the integration configuration.
2. Click **Save** and the server automatically assigns a port for the instance.

  Note: If you select **No Engine**, the listen port is assigned only after saving the configuration. If the instance is not saved, the server will assign a temporary port until the  instance starts running. Please make sure to click **Save**, not **Save & Exit**.
  If you configured the instance but exited before saving, you will need to create a new instance and follow the steps outlined above.

>   In XSOAR 8.9 version, the behavior for "No Engine" will be fixed, and the listen port will be assigned immediately upon configuration, eliminating the need to save the instance manually.

Reviewer: ```suggestion
  In Cortex XSOAR 8.9, the behavior for "No Engine" will change, and the listen port will be assigned immediately upon configuration, eliminating the need to save the instance manually.
```
